[SEO 14] Fix links to avoid redirects (#225)
KnapsackPro/docs.knapsackpro.com: _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md

## SimpleCov configuration for RSpec

> When you use [simplecov](https://github.com/simplecov-ruby/simplecov) gem in order to create test coverage for RSpec then you need to remember about one additional thing when you want to run tests in parallel on many CircleCI jobs. You set a unique name for the simplecov report with `SimpleCov.command_name`.

Reviewer: ```suggestion
When you use [simplecov](https://github.com/simplecov-ruby/simplecov) with RSpec, you have to set a unique name for its report with `SimpleCov.command_name`:
```
